pkg/oidc/oidc.go

```diff
@@ -0,0 +1,35 @@
+package oidc
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type WellKnownConfig struct {
+	Issuer                string   `json:"issuer"`
+	ScopesSupported       []string `json:"scopes_supported"`
+	AuthorizationEndpoint string   `json:"authorization_endpoint"`
+	TokenEndpoint         string   `json:"token_endpoint"`
+	JWKSetURI             string   `json:"jwks_uri"`
+}
+
+func LookupWellKnownConfig(issuerURL string) (*WellKnownConfig, error) {
+	resp, err := http.Get(fmt.Sprintf("%s/.well-known/openid-configuration", issuerURL))
```

Comment:
I'm getting this in my cluster when I try to use google. Do we need a way to trust a cert or did I do something wrong?:
`"Get https://accounts.google.com/.well-known/openid-configuration: x509: certificate signed by unknown authority"`